### PR TITLE
Polygon testnet

### PR DESCRIPTION
**Pull request by SuryaSan24x7**

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+    "semi": true,
+    "singleQuote": true,
+    "trailingComma": "es5",
+    "printWidth": 80,
+    "tabWidth": 2
+  }
+
```
Comment (nikakhachi) [inline .prettierrc]: You don't have to add this here. we already have `.prettierrc` in the `main` branch so remove it. And undo all the formatting changes.

File: deployedAddresses.txt [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+LiquidationManager deployed!:  0x5a5C0986Ae6AD6F3997489dADf213b3a4508b1C6
+DebtToken deployed!:  0x7c260AB5A478E30100Db35659326B8B30388D344
+BorrowerOperations deployed!:  0xC2fb75C45D2B28Cb817e109d81F35872D5bA270A
+StabilityPool deployed!:  0x17C039B55a32E98CCAfD39a5356F3e3305672159
+TroveManager deployed!:  0xc500905fA9f7f48bCA247696949796c9EeeebA42
+SortedTroves deployed!:  0xaE36c951C6C3fFFCDdCdE990AAB5056e722B09dc
+TokenLocker deployed!:  0xBA49C88f3d70A5d6A29E13B38ff7B05c636A7476
+IncentiveVoting deployed!:  0x8953752b6E7b1636F81C772f6f86662adc2C38fC
+BabelToken deployed!:  0x0A44D12C88fC0b551fb2208Bdb23F215BD347cC7
+BabelVault deployed!:  0x1cf11b3e7Af70341fCd21D5e17c6a5024D1e158d
```
Comment (nikakhachi) [inline deployedAddresses.txt:19]: Why is this file here?

File: hardhat.config.ts [hunk 3]
```diff
@@ -1,45 +1,62 @@
...
+      accounts: [
+        'ea5fbf909b888a1e810ef6ca42f06cb7eabc9a04095ec7e11fdeed7bb968ab07',
+      ],
+    },
+    scroll_testnet: {
+      url:'https://scroll-sepolia.drpc.org',
+      chainId:534351,
       accounts: [
-        "7c5b27c4f043051e405d03469e3f9dfe5b65df74376dcaf70db003d63a976efc",
+        'a0b8d309dd0afc95c94fc319f2de434f845a747d662c71ccbc08655c29299519',
```
Comment (nikakhachi) [inline hardhat.config.ts]: Why do you expose private keys here? You can use process.env.PRIVATE_KEY just like it's used in other places: `accounts: process.env.PRIVATE_KEY ? [process.env.PRIVATE_KEY] : []`

File: hardhat.config.ts [hunk 4]
```diff
@@ -1,45 +1,62 @@
...
+      gasPrice: 25000000000, // 25 gwei in wei
+    },
+    gate_testnet: {
+      url: 'https://meteora-evm.gatenode.cc',
+      chainId: 85,
+      accounts: [
+        'ea5fbf909b888a1e810ef6ca42f06cb7eabc9a04095ec7e11fdeed7bb968ab07',
+      ],
+    },
+    bsc_testnet: {
```
Comment (nikakhachi) [inline hardhat.config.ts]: 1 PR should include one new network. This PR shouldn't have any other changes then adding a polygon testnet

File: .prettierrc (hunk 1 above)
Comment (SuryaSan24x7) [reply .prettierrc]: ok
